src/modules/tools/endstops/Endstops.cpp

```diff
@@ -447,10 +447,15 @@ void Endstops::home_xy()
 {
     if(axis_to_home[X_AXIS] && axis_to_home[Y_AXIS]) {
         // Home XY first so as not to slow them down by homing Z at the same time
-        float delta[3] {alpha_max*2, beta_max*2, 0};
+        float min_distance= std::max(alpha_max*2, beta_max*2); //mimimum distance that will accomodate either axis
```

Comment:
well it's the minimum allowable distance, but it's the largest distance to go.   It's the minimum because I use it as a base guaranteeing that the axis with the furthest distance to go will have that much room.. it's a minimum because I modify the distances by possibly making them longer, not shorter.  I could probably come up with a better variable name